routes/foodTypeRouters.js

```diff
@@ -0,0 +1,23 @@
+/*
+** author: Fulin Shen
+** created on: May 21, 2016
+** description: see above
+*/
+
+var express = require('express');
+var router = express.Router();
+
+var trucks = require('../trucks');
+
+router.route('/')
+  .get(function(request, response) {
+    response.status(200).json(trucks.getFoodTypes());
```

Comment:
Since getFoodTypes will return an array, using send here will automatically set the content-type to application/json and the status to 200 for you. That said, what you did here still works fine - this is more of just an FYI.